added the min_peptide_score parameter to kojak

kfattila opened the pull request:
The new parameter is “min_peptide_score = #.#”. By default (i.e. if the parameter is not specified), the value is zero. This parameter is currently in the Kojak that ships with Crux, although at the time it was undocumented for testing. Over the past several months, I’ve found it valuable to set it between 0.2 and 0.5 for most datasets. I’m making it a documented parameter for all users later today.  -- MH

The range of this parameter is -10 .0till +10.0

I do not know how to use Kojak so the functionality of the parameter and that its value is passed to Kojak properly should be tested.

wsnoble commented:
Please add a line to the release notes say "Added  min_peptide_score parameter to Kojak."

kfattila commented:
I've changed the default Kojak parameter min_peptide_score to 0.1 as it is the default parameter in the Kojak program. I updated the smoke psm convert smoke-tests, they contain the new parameter. I also updated the release notes. Now, smoke tests pass.

bmx8177 commented:
This looks good to me. You can merge.

bmx8177 commented:
Though I just ran the performance test and the tide p-value line is now different. But I don't think this code has to do with that.

kfattila commented:
@bmx8177, The p-value curve in the performance test changed some time ago. the tide p.value curve actually got better, but unfortunately, the tide p.value +percolator curve a little bit shrinked. I will update the performance plots.
